omod/src/main/resources/config.xml

```diff
@@ -15,27 +15,19 @@
 	<require_version>${openMRSRuntimeVersion}</require_version>
 	<!-- / Module Properties -->
 
```

Comment:
Key meaningful change - advice is no longer loaded here, but conditionally, atom feed module is no longer required but optional